### PR TITLE
Ogc conformance testing

### PR DESCRIPTION
**Pull request by mhugent**
This PR adds a quick tutorial how to setup the OGC confomance tests for WMS 1.3.0 and WMS 1.1.1

### CODE REVIEW

File: source/site/getinvolved/development/qgisdevelopersguide/ogcconformancetesting.rst [hunk 1]
```diff
@@ -0,0 +1,93 @@
+
```
Comment (yjacolin) [inline source/site/getinvolved/development/qgisdevelopersguide/ogcconformancetesting.rst:1]: could you add some index see http://docs.qgis.org/testing/en/docs/documentation_guidelines/writing.html#index

File: source/site/getinvolved/development/qgisdevelopersguide/ogcconformancetesting.rst [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+OGC Conformance Testing
+=======================
+
+
+The Open Geospatial Consotrium (OGC) provides tests which can be run free of charge to make sure a server is compliant with a certain specification. This chapter provides a quick tutorial to setup the WMS tests on an Ubuntu system. A detailed documentation can be found at the OGC website (http://www.opengeospatial.org/compliance).   
+
+Setup of WMS 1.3 and WMS 1.1.1 conformance tests
+------------------------------------------------
+
+sudo apt-get install openjdk-8-jdk
```
Comment (yjacolin) [inline source/site/getinvolved/development/qgisdevelopersguide/ogcconformancetesting.rst]: use .. code:: shell

File: source/site/getinvolved/development/qgisdevelopersguide/ogcconformancetesting.rst [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+mvn install
+
+
+Test project
+------------
+
+For the WMS tests, data can be downloaded and loaded into a QGIS project:
+
+wget http://cite.opengeospatial.org/teamengine/about/wms/1.3.0/site/data-wms-1.3.0.zip
```
Comment (yjacolin) [inline source/site/getinvolved/development/qgisdevelopersguide/ogcconformancetesting.rst]: .. code:: shell

File: source/site/getinvolved/development/qgisdevelopersguide/ogcconformancetesting.rst [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+unzip data-wms-1.3.0.zip
+
+Then create a QGIS project according to the description in http://cite.opengeospatial.org/teamengine/about/wms/1.3.0/site/. To run the tests, we need to provide the GetCapabilities URL of the service later.
+
+
+Running the WMS 1.3.0 test
+--------------------------
+
+export PATH=/usr/lib/jvm/java-8-openjdk-amd64/bin:$PATH
```
Comment (yjacolin) [inline source/site/getinvolved/development/qgisdevelopersguide/ogcconformancetesting.rst]: .. code:: shell

File: source/site/getinvolved/development/qgisdevelopersguide/ogcconformancetesting.rst [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+
+cd ~/te-install
+
+./bin/unix/test.sh -source=$HOME/src/ets-wms13/src/main/scripts/ctl/main.xml
+
+
+Running the WMS 1.1.1 test
+--------------------------
+
+export PATH=/usr/lib/jvm/java-8-openjdk-amd64/bin:$PATH
```
Comment (yjacolin) [inline source/site/getinvolved/development/qgisdevelopersguide/ogcconformancetesting.rst]: .. code:: shell

**Review by yjacolin — CHANGES_REQUESTED** [review]

**Comment by yjacolin** [comment]
thanks @mhugent!

File: source/site/getinvolved/development/qgisdevelopersguide/ogcconformancetesting.rst [hunk 6]
```diff
@@ -0,0 +1,93 @@
+
+OGC Conformance Testing
+=======================
+
+
+The Open Geospatial Consotrium (OGC) provides tests which can be run free of charge to make sure a server is compliant with a certain specification. This chapter provides a quick tutorial to setup the WMS tests on an Ubuntu system. A detailed documentation can be found at the OGC website (http://www.opengeospatial.org/compliance).
```
Comment (DelazJ) [inline source/site/getinvolved/development/qgisdevelopersguide/ogcconformancetesting.rst]: _Typo_: Consortium
_Proposal_: use a hyperlinked text, namely \`OGC website \<http://www.opengeospatial.org/compliance>`_

To ease reading and review, lines should not be over 80 characters

File: source/site/getinvolved/development/qgisdevelopersguide/ogcconformancetesting.rst (hunk 2 above)
Comment (DelazJ) [reply source/site/getinvolved/development/qgisdevelopersguide/ogcconformancetesting.rst]: If you go with the code block, then you can get rid of empty lines used between two following commands inside the block.

File: source/site/getinvolved/development/qgisdevelopersguide/ogcconformancetesting.rst [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+export TE_BASE=~/TE_BASE
+
+unzip ./teamengine-console/target/teamengine-console-4.11-SNAPSHOT-base.zip -d $TE_BASE
+
+mkdir ~/te-install
+
+unzip ./teamengine-console/target/teamengine-console-4.11-SNAPSHOT-bin.zip -d ~/te-install
+
+
+#Download and install WMS 1.3.0 test
```
Comment (DelazJ) [inline source/site/getinvolved/development/qgisdevelopersguide/ogcconformancetesting.rst]: What does the # character stand for? A comment inside the code or numbering or something else...

**Review by yjacolin — APPROVED** [review]

**Comment by rldhont** [comment]
@mhugent cool to see this. Do you have the QGIS project ? To run test ?

**Comment by mhugent** [comment]
@rldhont: the project does not involve styling at all. Just download the zip file with the data and load all the shapefiles and geotiffs into a project (the layer names should already be okay). Then you can run the tests against that project.

**Comment by rldhont** [comment]
@mhugent ok thanks!

**Pull request merged**